packages/manager/modules/pci/src/projects/project/private-registry/translations/Messages_fr_FR.json

```
@@ -44,7 +44,7 @@
   "private_registry_update_name_label": "Choisir un nom pour le Registre privé",
   "private_registry_update_success": "Le nom de Registre privé a été renommé en {{ newRegistryName }}",
   "private_registry_update_error": "Une erreur s'est produite lors du changement de nom du Registre privé {{ registryName }}: {{ message }}",
-  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Veuillez générer des informations d'identification pour accéder à votre Registre",
+  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Vous pouvez générer des informations d'identification pour accéder à votre Registre une fois qu'il est créé.",
```

```suggestion
  "private_registry_onboarding_success": "Votre registre privé {{ registryName }} a été créé. Vous pourrez générer des informations d'identification pour accéder à votre Registre une fois qu'il sera créé.",
```

"Vous pouvez ... une fois qu'il est créé" : "You can ... once it's created"
"Vous pourrez ... une fois qu'il sera créé" : "You will be able ... once it will be created"